assets/i18n/en.json

```diff
@@ -99,6 +99,7 @@
     "I18N_BLOG_POST_THUMBNAIL_PICTURE_UPLOAD": "Add a thumbnail",
     "I18N_BLOG_POST_UNTITLED_HEADING": "Untitled",
     "I18N_CARD_HEIGHT_LIMIT_WARNING_MESSAGE": "This card is quite long, and students might lose interest. Consider shortening it, or breaking it into two cards.",
+    "I18N_CARD_CONTENT_LIMIT_WARNING_MESSAGE": "This card content is too long. Please keep it below 4500 characters to save.",
```

Comment:
Can we change this to be ERROR_MESSAGE because it is more strict than a warning? What do you think?

Reply:
Done